server/vehicle-persistence.lua

```diff
@@ -1,3 +1,6 @@
+local persistence = GetConvarInt('qbx:enableVehiclePersistence', 0)
```

Comment:
FYI, values `"true"` and `"false"` are interpreted as `1` and `0` respectively by `GetConvarInt`. But at this point this can actually be replaced by the new `GetConvarBool` I believe, which is available since artifact 10543.